src/templates/speciality.js

```diff
@@ -58,17 +62,16 @@ const Speciality = ({
         title={title}
       />
       <EventSection events={events} title={title} eventIcon={eventIcon} />
-      <TalksSection
-        talks={getExternalType(speciality, `Talk`)}
-        videoIcon={videoIcon}
-      />
+      <TalksSection talks={talks} videoIcon={videoIcon} />
       <BlogListing
         title="Blog posts"
         description={`${title} articles created by members of YLD for the community.`}
         posts={posts}
       />
-      <TutorialsSection tutorials={getExternalType(speciality, `Tutorial`)} />
-      <BooksSection title={title} books={getExternalType(speciality, `Book`)} />
+      {tutorials.length > 0 ? (
```

Comment:
need to check first that tutorials is no undefined otherwise we could be seeing `cannot read property length of undefined`

`{tutorials && tutorials.length && ....}`

Reply:
Because of [this line](https://github.com/yldio/yld.io/pull/197/files/2a9681fb0e7255d247ddafe577eb4b2e0ea2caef#diff-6a7ed84b5bdb2c4809423a71f05a42eeL20) there will actually always be an empty array. But you're right, probably a good idea to code more defensively in general.

Thanks :+1:

Reply:
Ok, so there is already a safety net with the line your mentioned.
One minor change I'd suggest. You don't need to compare to 0.
`tutorials.length ? (` is enough for your ternary

Reply:
Yeah it's true. I just think `tutorials.length > 0` reads nicely :laughing: